PROJECT

```diff
@@ -1,8 +1,18 @@
-domain: openshift.io
-layout: go.kubebuilder.io/v2
-projectName: managed-fusion-agent
-repo: github.com/red-hat-storage/managed-fusion-agent
-version: "3"
+domain: ibm.com
+layout:
+- go.kubebuilder.io/v2
 plugins:
   manifests.sdk.operatorframework.io/v2: {}
   scorecard.sdk.operatorframework.io/v2: {}
+projectName: managed-fusion-agent
+repo: github.com/red-hat-storage/managed-fusion-agent
+resources:
+- api:
+    crdVersion: v1beta1
```

Comment:
- generated files say `v1alpha1` and this says `v1beta1`, anything missed 🤔 ?

Reply:
That's the CRD version, it is the k8s version that is used to scaffold CRD.

The ManagedFusionOffering CR will have apiVersion as misf.ibm.com/v1alpha1 (this comes from https://github.com/red-hat-storage/managed-fusion-agent/pull/8/files#diff-d1fb402f1269ba7396dca9cbd03b44ac055874669bce1d0f239fa2f54ea3584bR17)